cpa/parsing.go

```diff
@@ -139,3 +141,36 @@ func (err MultiError) Error() string {
 		return fmt.Sprintf("%d error(s) occurred: %s", len(err), strings.Join(messages, "; "))
 	}
 }
+
+//LoadPolicyFile takes policy file path as an input, and returns parsed policy
+func LoadPolicyFile(filePath string) (*Policy, error) {
+	documentBundle := map[string]string{}
+	fileContent, err := ioutil.ReadFile(filepath.Clean(filePath))
```

Comment:
lets use the OS package instead of the deprecated ioutil package.
This was an issue with the CLI but everything is being updated to use go1.18 so this isn't an issue anymore.